ViewEnvironment/Sources/EnvironmentValues+ViewEnvironment.swift

```diff
@@ -14,16 +14,19 @@
  * limitations under the License.
  */
 
-import SwiftUI
-import WorkflowUI
+#if canImport(SwiftUI)
 
-private struct ViewEnvironmentKey: EnvironmentKey {
-    static let defaultValue: ViewEnvironment = .empty
-}
+import SwiftUI
 
-public extension EnvironmentValues {
```

Comment:
hazy questions: will this impact clients that currently depend on both these targets? will the experimental library need to be bumped so it doesn't conflict with the newer ViewEnvironment implementation?

Reply:
> will this impact clients that currently depend on both these targets?

It seems unlikely since you'd already need to `import ViewEnvironment` to work with the type of this environment value.

I think we should consider adding a `@_exported import ViewEnvironment` to `WorkflowSwiftUIExperimental`.

AFAIK the only client right now is `MarketSwiftUI` and `SquareDebugMenu` which I'm working with @square-tomb on and am happy to resolve an issues that come up in either repo as a result of this change! `MarketSwiftUIExperimental` is still experimental so changes like this should be expected (right?).

> will the experimental library need to be bumped so it doesn't conflict with the newer ViewEnvironment implementation?

Are these separate version numbers/releases?

I do think we'll want to bump both of them and update consumers of these frameworks together.

Reply:
> Are these separate version numbers/releases?

Yep; details on that [here](https://github.com/square/workflow-swift/blob/6866c752078fc5edabda5f2f971cbafd8fbe21d2/WorkflowSwiftUIExperimental/README.md#versioning).

Reply:
thanks, that makes sense. because the definition of the extension moved but kept the same name, was just thinking whether both libraries would need a version bump to avoid ambiguous symbols (seems like they will, though perhaps it could be worked around with a module prefix).